### PR TITLE
teracopy and simutrans cleaners

### PR DESCRIPTION
**Pull request by mikevainio**
currently only windows

### CODE REVIEW

File: release/teracopy.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  </option>
+  <option id="log">
+    <label>Transfer history log</label>
+    <description>Deletes the history log.</description>
+    <action command="delete" search="glob" path="$APPDATA\TeraCopy\Transfer.log"/>
+  </option>
+  <option id="last">
+    <label>Last copied list</label>
+    <description>Deletes the list.</description>
+    <action command="delete" search="glob" path="$APPDATA\TeraCopy\FileList.dat"/>
```
Comment (az0) [inline release/teracopy.xml]: Here "glob" should be "file" because there are no special glob patterns like *

File: release/teracopy.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <description>File copier manager</description>
+  <option id="lastdir">
+    <label>Last Target Folder</label>
+    <description>Deletes the last target folder path used.</description>
+    <action command="winreg" path="HKCU\Software\Code Sector\TeraCopy" name="LastTargetFolder"/>
+  </option>
+  <option id="log">
+    <label>Transfer history log</label>
+    <description>Deletes the history log.</description>
+    <action command="delete" search="glob" path="$APPDATA\TeraCopy\Transfer.log"/>
```
Comment (az0) [inline release/teracopy.xml]: Here "glob" should be "file" because there are no special glob patterns like *

File: release/simutrans.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<!--
+
+    BleachBit cleaner for
+    Simutrans v111 (by Simutrans Team) and compatible versions
+    
+    This CleanerML mark-up is intended for use with the BleachBit program
+    (available at http://bleachbit.sourceforge.net) and was developed by
+    Erratalog blog (http://erratalog.blogspot.com). The mark-up may be
+    modified and/or redistributed so long as the Erratalog blog is cited
+    as a source for the mark-up or derivative works developed from it.
```
Comment (az0) [inline release/simutrans.xml]: I understand you should have credit for this. However, this creates a license that I think is not compatible with the GNU General License, which is the license for this repository: please see https://github.com/az0/cleanerml#license .

You could instead do something like this, and there is no reason anyone would really alter it:

````This cleaner Copyright (c) 2015 by Mike Vainio (http://erratalog.blogspot.com)````

Comment (mikevainio) [inline release/simutrans.xml]: I agree, and only realized this after saving all your documentation
and reading it at home last night. I re-coded the mark-up for those
two cleaners as well as a number of additional cleaners. All will be
uploaded momentarily. The mark-up will now be identical to those you
released, plus the following three lines of text:
<---...
...your comments...

CleanerML   : Akelpad v4.7.9 (by Kuznetsov and Aleksander)
Contributor   : Erratalog
Web site      : http://erratalog.blogspot.com
-->

**Pull request closed without merging**